### PR TITLE
Update main README

### PR DESCRIPTION
**Pull request by matsduf**
Updated with which versions of OS, databases, Perl and browsers that Zonemaster supports.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,9 +50,11 @@ architecture listed below.
 ### Supported operating system versions
 
 * CentOS 7
+* CentOS 8
 * Debian 8
 * Debian 9
-* FreeBSD 11.2
+* Debian 10
+* FreeBSD 11.3
```
Comment (mattias-p) [inline README.md]: That's a lot of OS versions!

File: README.md [hunk 2]
```diff
@@ -84,19 +98,21 @@ tested with the default version of Perl in the OSs as listed in the table above.
...
 
 Operating System | Browser | Version
 ---------------- | ------- | -------
-Ubuntu 18.04     | Firefox | 64, 65, 66
-Ubuntu 18.04     | Chrome  | 66
-Windows 10       | Firefox | 64, 65, 66
-Windows 10       | Chrome  | 73
-MacOs            | Firefox | 65
-MacOs            | Chrome  | 73
+Ubuntu 18.04     | Firefox | 69
```
Comment (mattias-p) [inline README.md]: Why did we have several versions before?

File: README.md [hunk 3]
```diff
@@ -178,6 +194,8 @@ repository (i.e. [general issues in Zonemaster](https://github.com/zonemaster/zo
 
 ## Notable bugs and issues
 
+*TO BE UPDATED AT RELEASE*
+
```
Comment (mattias-p) [inline README.md]: There's a step for this in the release process. I don't think we need to spell it out in the document.

File: README.md (hunk 1 above)
Comment (matsduf) [reply README.md]: Yes, really!

File: README.md (hunk 2 above)
Comment (matsduf) [reply README.md]: I do not know. Now I used the information from the Browser vendors.

File: README.md (hunk 3 above)
Comment (matsduf) [reply README.md]: I just wanted to have something sticking in the eyes when looking at the document in the develop branch.

**Pull request merged**